diff --git a/node_modules/@emotion/server/create-instance/dist/emotion-server-create-instance.cjs.dev.js b/node_modules/@emotion/server/create-instance/dist/emotion-server-create-instance.cjs.dev.js
index 530fa0a..388bf86 100644
--- a/node_modules/@emotion/server/create-instance/dist/emotion-server-create-instance.cjs.dev.js
+++ b/node_modules/@emotion/server/create-instance/dist/emotion-server-create-instance.cjs.dev.js
@@ -2,9 +2,9 @@
 
 Object.defineProperty(exports, '__esModule', { value: true });
 
-var through = require('through');
-var tokenize = require('html-tokenize');
-var pipe = require('multipipe');
+var through = () => {};
+var tokenize = () => {};
+var pipe = () => {};
 
 function _interopDefault (e) { return e && e.__esModule ? e : { 'default': e }; }
 
diff --git a/node_modules/@emotion/server/create-instance/dist/emotion-server-create-instance.cjs.prod.js b/node_modules/@emotion/server/create-instance/dist/emotion-server-create-instance.cjs.prod.js
index 4721b22..77531ac 100644
--- a/node_modules/@emotion/server/create-instance/dist/emotion-server-create-instance.cjs.prod.js
+++ b/node_modules/@emotion/server/create-instance/dist/emotion-server-create-instance.cjs.prod.js
@@ -4,7 +4,7 @@ Object.defineProperty(exports, "__esModule", {
   value: !0
 });
 
-var through = require("through"), tokenize = require("html-tokenize"), pipe = require("multipipe");
+var through = () => {}, tokenize = () => {}, pipe = () => {};
 
 function _interopDefault(e) {
   return e && e.__esModule ? e : {
